server/builtin/install.go

```diff
@@ -35,6 +37,8 @@ func (a *builtinApp) installCommandBinding(loc *i18n.Localizer) apps.Binding {
 			},
 		}
 	}
+
+	// install http|listed
```

Comment:
Why `listed`? Isn't that what the previous case is about?

Reply:
the first use case (cloud) does not need a subcommand since it only exposes the listed apps, thus `/apps install appid`. The second use-case differentiates between `/apps install listed appid`, identical handler to the first one, and `/apps install http url`. The latter allows to install any (allowed) app deployment mode, from an HTTP manifest.

4/5 that this all needs a refactoring into a few simple tables, but not ATM?